Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/nukiemothroach.yml

```diff
@@ -0,0 +1,168 @@
+﻿- type: entity
+  name: nuclear operative mothroach
+  parent: MobCockroach
```

Comment:
Why is this parented to Cockroach?

You can cut down the components to like... three or four by not parenting it to the cockroach